### PR TITLE
MS. Lat. th. d. 13: Revise description

### PR DESCRIPTION
**Pull request by adunning**

### CODE REVIEW

**Comment by adunning** [comment]
This is ready for review now. Not sure if it's worth creating authorities for these works (not in VIAF) based on <https://doi.org/10.1484/J.BPM.3.595> – right now they have zeroed out IDs.

Also needed is an entry for `person_5320354`, and the accompanying work description should also be updated to identify its author.

File: collections/Lat_th/MS_Lat_th_d_13.xml [hunk 1]
```diff
@@ -49,151 +42,147 @@
...
+                  </bindingDesc>
                </physDesc>
                <history>
-                  <provenance>'<orgName role="fmo" key="org_415145858126223022295">
-                        Cartusiae Erford
-                     </orgName>' on 1st fol. &amp; top edge, 16th century</provenance>
-                  <provenance>Bought, 1929.</provenance>
+                  <origin>
+                     <origPlace>
+                        <country key="place_7000084">German</country>, <settlement key="place_7024031">Erfurt</settlement>, <orgName role="fmo" key="org_415145858126223022295">Charterhouse of Salvatorberg</orgName></origPlace>: inscribed, <q>Ad Carthusiam Erford</q> (head fore-edge, 16th century?); <q>Cartusiæ Erford</q> (fol. 1r, 17th or 18th century). Some booklets are autographs of <persName key="person_88951828" role="fmo">Ioannes Hagen de Indagine</persName>, dated 1457, 1458, 1459, 1460, 1468. These correspond to his periods of activity at Erfurt: 1440–54, 1457–60, and 1467–1475×76. Examples of his script also appear in MS. Hamilton 46, MS. Hamilton 48, MS. Hamilton 49, MS. Hamilton 54, and MS. Hamilton 57.</origin>
```
Comment (holfordm) [inline collections/Lat_th/MS_Lat_th_d_13.xml:65]: I think origin and provenance should be separated here - yes, written at the charterhouse (at least in part) s. xv 3/4, but then also owned there until its dissolution.

File: collections/Lat_th/MS_Lat_th_d_13.xml [hunk 2]
```diff
@@ -49,151 +42,147 @@
...
+                  <provenance when="1878-11-06"><orgName key="org_124303338">Messrs. Puttick and Simpson</orgName>, 6 November 1878, lot no. 284. <!-- TODO: the BL have an annotated copy; check to see if it identifies purchaser --> Sale catalogue listing pasted to an endleaf alongside a clipping from an unidentified newspaper on the <q>Death warrant of Christ</q>, a piece that ran in several American publications in 1866.</provenance>
+                  <acquisition when="1929">Purchased, 1929: notice in <title>Oxford University Gazette</title> (5 March 1930), p. 397.</acquisition>
                </history>
-               <additional>
-                  <adminInfo>
-                     <recordHist>
-                        <source>Typescript description by Bodleian library staff</source>
-                     </recordHist>
-                  </adminInfo>
-               </additional>
```
Comment (holfordm) [inline collections/Lat_th/MS_Lat_th_d_13.xml:84]: did you replace this section with an updated one?

File: collections/Lat_th/MS_Lat_th_d_13.xml [hunk 3]
```diff
@@ -49,151 +42,147 @@
...
+                  <provenance when="1878-11-06"><orgName key="org_124303338">Messrs. Puttick and Simpson</orgName>, 6 November 1878, lot no. 284. <!-- TODO: the BL have an annotated copy; check to see if it identifies purchaser --> Sale catalogue listing pasted to an endleaf alongside a clipping from an unidentified newspaper on the <q>Death warrant of Christ</q>, a piece that ran in several American publications in 1866.</provenance>
+                  <acquisition when="1929">Purchased, 1929: notice in <title>Oxford University Gazette</title> (5 March 1930), p. 397.</acquisition>
                </history>
-               <additional>
-                  <adminInfo>
-                     <recordHist>
-                        <source>Typescript description by Bodleian library staff</source>
-                     </recordHist>
-                  </adminInfo>
```
Comment (holfordm) [inline collections/Lat_th/MS_Lat_th_d_13.xml:83]: useful to add bibliographical ref to Watson D&D

**Pull request merged**